login-workflow/docs/language-support.md

```diff
@@ -8,6 +8,101 @@ This package supports translations to different languages using [i18next](https:
 -   Spanish
 -   Simplified Chinese
 
```

Comment:
We need some sort of introduction here that explains that the workflow dictionaries are separate/independent of the user application dictionaries.

Reply:
Added introduction, please check.